main.go

```diff
@@ -38,8 +38,9 @@ var (
 	excludes = app.Flag("exclude", "Resource sets to exclude explicitly").Short('e').Strings()
 
 	// Commands
-	template     = app.Command("template", "Template resource sets and print them")
-	templateFile = template.Arg("file", "Cluster configuration file to use").Required().String()
+	template          = app.Command("template", "Template resource sets and print them")
+	templateFile      = template.Arg("file", "Cluster configuration file to use").Required().String()
+	templateOutputDir = template.Flag("output-dir", "Output directory where to place templated files").Short('o').String()
```

Comment:
If we only support a single type of output (i.e. "directory"), the flag itself could probably just be named `output`.

Reply:
Referring to my comment from below: A better help string may be something like

> Output directory in which to save templated files instead of printing them